lib/facility_maker.rb

```diff
@@ -0,0 +1,50 @@
+class FacilityMaker
+  attr_reader :add_facility, :facility, :add_facility, :facility_info, :name
+  attr_accessor :name, :address, :phone, :facilities, :add_facility
+
+  def new_facility(state_facilities)
+    state_facilities.map do |state_facility|
+      gen_facility = Facility.new({name: new_facility_name(state_facility), address: new_facility_address(state_facility), 
+      city: state_facility[:city], state: state_facility[:state], zipcode: new_facility_zipcode(state_facility), 
+      phone: new_facility_phone(state_facility)})
+    end
+  end
+
+  def new_facility_name(state_facility)
+    if state_facility[:website] && state_facility[:website].include?("oregon")
```

Comment:
It's really smart that you are checking both `state_facility[:website]` and 
`state_facility[:website].include?("oregon")`. It helps us catch bad data and avoid our program from returning errors.